lib/libthr/thread/thr_attr.c

```diff
@@ -375,7 +375,16 @@ _pthread_attr_init(pthread_attr_t *attr)
 	else {
 		/* Initialise the attribute object with the defaults: */
 		memcpy(pattr, &_pthread_attr_default, sizeof(struct pthread_attr));
-
+#ifdef __CHERI_PURE_CAPABILITY__
```

Comment:
I think we do for now we do because we'll need to extend the mips `thr_new()` interface to support this and we haven't thought about managing hybrid syscalls yet.  That's going to require some significant thinking.